AdvUtils/Logger.cs

```diff
@@ -119,23 +119,27 @@ public static void WriteLine(Level level, ConsoleColor color, string s, params o
                 Console.ForegroundColor = color;
 
                 StringBuilder sb = new StringBuilder();
-                sb.AppendFormat("{0},{1} ", level.ToString(), DateTime.Now.ToString());
 
-                if (args.Length == 0)
-                    sb.Append(s);
-                else
-                    sb.AppendFormat(s, args);
+                if (s != "")
```

Comment:
String.IsNullOrEmpty(String) woudl be better. It indicates whether the specified string is null or an empty string ("").